docs/explanations/modeling_extensions/surrogate/alamopy/alamopy-cli.rst

```diff
@@ -14,32 +14,57 @@ Basic ALAMOPY.ALAMO options
 Data Arguments
```

Comment:
Is this actually correct? I believe the command line interface only applies to the old ALAMO wrapper (and this doc is explicitly discussing a cli). If that is true, then we should just remove this documentation as it is no longer supported (also, the changes here would not be correct as the old API would still apply).